README.md

```diff
@@ -54,6 +54,9 @@ Read our [privacy statement](https://go.microsoft.com/fwlink/?LinkId=521839) to
 Telemetry is disabled in development builds by default, and can be enabled with the `SEND_TELEMETRY`
 build flag.
 
+## Currency Converter
+Windows Calculator includes a currency converter feature that uses mock data in developer builds.  The data that Microsoft uses for the currency converter feature (e.g., in the retail version of the application) is not licensed for your use.  The mock data will be clearly identifiable as it references planets instead of countries.
```

Comment:
Please try to keep individual lines in the markdown to no longer than 100-120 characters.  Force a line-break after that.  It will still render just fine, but becomes easier to read (no horizontal scrolling) for those looking at the raw file.

Additionally, now that I see that the mock data is completely static regardless of the `GET` parameter, I would suggest this slightly altered statement (or some similar change):

```suggestion
Windows Calculator includes a currency converter feature that uses mock data in developer builds.  The data that Microsoft uses for the currency converter feature (e.g., in the retail version of the application) is not licensed for your use.  The mock data will be clearly identifiable as it references planets instead of countries, and remains static regardless of selected inputs.
```